src/getHumanAge.test.js

```diff
@@ -1,8 +1,70 @@
+'use strict';
+
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
   test('should be declared', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an Array', () => {
+    const result = getHumanAge(10, 10);
+
+    expect(result).toBeInstanceOf(Array);
+  });
+
+  test('should return zeros if there are zeros in arguments', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return integer numbers', () => {
+    expect(getHumanAge(100, 100)).toEqual([21, 17]);
+  });
+
+  test('should return with exactly 2 elements', () => {
+    const result = getHumanAge(15, 15);
+
+    expect(result.length).toBe(2);
+  });
+
+  test('should return 0 if the argument is less than 15', () => {
```

Comment:
```suggestion
  test('should return [0, 0] if the argument is less than 15', () => {
```
because now it seems like that the function returns a number, but it's not true. same to others